tests/integration/pluggable_integration_test.go

```diff
@@ -51,11 +53,9 @@ func TestPlugins_CatchStderr(t *testing.T) {
 			},
 		}}
 
-		createInternalPlugin := func(string, interface{}) (plugins.Plugin, error) {
-			return nil, nil
-		}
-		ll := NewPluginLoader(c.Config, createInternalPlugin)
-		_, _, err = ll.Load(cfg)
+		ll := pluggable.NewPluginLoader(c.Config)
+		conn, err := ll.Load(context.Background(), cfg)
+		conn.Close()
```

Comment:
Should this be in a defer instead?

Reply:
We aren't doing anything with the connection in the test so there's no need to put off closing the connection.